bin/commands/runs.js

```diff
@@ -77,9 +77,13 @@ module.exports = function run(args) {
           // Create build
 
           //setup Local Testing
+          let options = {};
+          let startTime = new Date().getTime();
           let bs_local = await utils.setupLocalTesting(bsConfig, args);
```

Comment:
This will be returning [a promise](https://github.com/browserstack/browserstack-cypress-cli/blob/master/bin/helpers/utils.js#L445) so checking the start and end time and taking the diff will not really help. We should check for [actual start time](https://github.com/browserstack/browserstack-cypress-cli/blob/master/bin/helpers/utils.js#L450-L455) for loggin.